dnf/conf/substitutions.py

```diff
@@ -26,9 +26,10 @@
 
 class Substitutions(dict):
 
-    def __init__(self):
+    def __init__(self, arch=None):
```

Comment:
We should do it better way. I don't like that arch has a privilege to be passed in the constructor.